packages/node-integration-tests/suites/tracing/prisma-orm/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "sentry-prisma-test",
```

Comment:
let's move this out into another PR so we can discuss the integration test separately? In the mean time, we can add sanity check unit tests to this PR (similar to what we have for mongo for ex.)